modules/aws/variables.tf

```
@@ -133,3 +133,9 @@ variable "write_policy_files" {
   description = "Write the policy files locally to disk for debugging and validation"
   type        = bool
 }
+
+variable "vendor_federation" {
```

Would it be better to name it `allow_vendor_federation` to keep consist?
```suggestion
variable "allow_vendor_federation" {
```

---

This is fine, I'll make this change.